IRremoteESP8266.cpp

```diff
@@ -702,23 +702,32 @@ static void ICACHE_RAM_ATTR read_timeout(void *arg __attribute__((unused))) {
 
 static void ICACHE_RAM_ATTR gpio_intr() {
   uint32_t gpio_status = GPIO_REG_READ(GPIO_STATUS_ADDRESS);
+  uint32_t now = system_get_time();
```

Comment:
If it changes the delays noticeably, then keep it as system_get_time(), and add a comment as for why so someone doesn't have the same discussion again. ;-)

If it is negligible, lets go with cleaner/consistent code.